device-registry/src/main/scala/org/genivi/sota/device_registry/DeviceRepository.scala

```diff
@@ -75,9 +75,29 @@ object Devices {
     def pk = primaryKey("id", id)
   }
 
+  type GroupInfoType = Json
+  case class GroupInfo(groupName: String, namespace: Namespace, groupInfo: GroupInfoType)
```

Comment:
This should have it's own repo. (We should have created one also for SystemInfo but we didn't, but we shouldn't make the same mistake twice)